File: insercionarray/inser3/inser.js
let inputArr = [11, 21, 31, 41, 51, 61];
let ele = 52;

let antes = "Antes de la insercion, el array es: ";
for (let i = 0; i < inputArr.length; i++) {
    antes += inputArr[i] + " ";
}
console.log(antes);

inputArr.push(ele);

let despues = "Despues de la insercion, el array es: ";
for (let i = 0; i < inputArr.length; i++) {
    despues += inputArr[i] + " ";
}
console.log(despues);

File: insercionarray/inser3/inser.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class inser {
    public static void main(String[] args) {
        List<Integer> inputArr = new ArrayList<>(Arrays.asList(11, 21, 31, 41, 51, 61));
        int num = 52;

        System.out.print("Antes de la insercion, el array es: ");
        for (int i = 0; i < inputArr.size(); i++) {
            System.out.print(inputArr.get(i) + " ");
        }

        inputArr.add(num);

        System.out.print("\nDespues de la insercion, el array es: ");
        for (int i = 0; i < inputArr.size(); i++) {
            System.out.print(inputArr.get(i) + " ");
        }
    }
}